app/src/main/res/values-ru/strings.xml

```diff
@@ -105,4 +105,12 @@
     <string name="stats_exp_for_a_week">опыта за неделю</string>
     <string name="ex_dialog_continue">Продолжить</string>
     <string name="tutorial">Обучение</string>
+    <string name="streak_success">Серия из %d ответов</string>
+    <string name="streak_failed">Серия потеряна</string>
+    <string name="streak_restored">Серия восстановлена</string>
+    <string name="use_coupon">Использовать</string>
+    <string name="streak_restore_text">Купоны позволяют не потерять бонус к опыту за серию правильных ответов, если вы ответили неверно. Вам доступно:</string>
```

Comment:
Please, combine with the 2nd part and make 1 meaningful string. (https://medium.com/google-developer-experts/android-strings-xml-things-to-remember-c155025bb8bb Format section)